src/avrdude.1

```diff
@@ -371,6 +372,13 @@ or
 .Pa ${HOME}/.avrduderc
 file to assign a default value to keep from having to specify this
 option on every invocation.
+.Pp
+Note that some official Microchip programmers stores the bitclock setting and
+will continue to use it until a different value is provided. This applies to
+"2nd gen" programmers (AVRISPmkII, AVR Dragon, JTAG ICE mkII, STK600) and
+"3rd gen"programmers (JTAGICE3, Atmel ICE, Power Debugger). "4th gen"
+programmers (PICkit 4, MPLAB SNAP) will store the last user-specified bitclock
+until it's disconnected from the computer.
```

Comment:
Small suggestions:
programmers stores -> programmers store
it's disconnected -> the programmer is disconnected